lib/developer_portal/app/controllers/developer_portal/api_docs/services_controller.rb

```diff
@@ -1,5 +1,4 @@
 class DeveloperPortal::ApiDocs::ServicesController < DeveloperPortal::BaseController
-  skip_before_action :set_x_content_type_options_header
```

Comment:
https://github.com/3scale/porta/pull/1551/files#r378172640

Reply:
How is it solved?

Reply:
we now have a before action that manually disable x_frame header. https://github.com/3scale/porta/pull/1551/files#diff-1862ca4b36d9904f13fb697bb4daab9eR40

Reply:
ah.. I will disable this also.. only disabled the x_frame

Reply:
disabled with a before_action